pyproject.toml

```
@@ -150,6 +153,22 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
+current_version = "2023.3.0"
+version_pattern = "YYYY.MM.INC0[-TAG]"
+commit_message = """
+Bump version to {new_version}
+"""
+
+
+[tool.bumpver.file_patterns]
+"pyproject.toml" = [
+    'current_version = "{version}"',
+]
+"integreat_compass/__init__.py" = [
+    '__version__ = "{version}"',
+]
+
```

```suggestion
"docs/src/conf.py" = [
    'release = "{version}"',
]

```
Could you also fix the version in this file from `2023.4.0` to `2023.3.0`?

---

This should be applied now